addons/compat_spe/compat_spe_csw/functions/fn_alter_height.sqf

```diff
@@ -0,0 +1,103 @@
+#include "..\script_component.hpp"
+/*
+Author: Kerc, Original by www.3commandobrigade.com, edited by johnb43 to be made CSW compatible
+Remove ammo from static weapon
```

Comment:
```suggestion
Alter weapon height
```

Reply:
That's the original header - do we want to change it or leave it as much as it was originally?

If we change it, then I think I'll optimise the functions (e.g. use lazy evals).